### ISSUE TITLE
`4.1.0` has no longer support Angular 12

### ISSUE THREAD

**Issue by begandroide** [tanansatpal/ngx-mat-intl-tel-input#134]
Hi! Today I upgrade `ngx-mat-intl-tel-input` from `4.0.0` to `4.1.0` and the application not compile because a lot of errors.

![image](https://user-images.githubusercontent.com/22018004/141297668-a07341b8-1476-408a-8519-06943eb2dade.png)

**Comment by venraij** [comment]
I'm using the package in an angular 12 project as well, and it has no issues at all. Have you tried deleting your node modules and installing them all new again?

**Comment by begandroide** [comment]
@venraij thanks you for response. Yes, I deleted node_modules folder and install again and same error, we found the problem in CI pipeline, where install is in cleaned up folder. So I downgrade to `4.0.0` and all works fine again. I'm going to check again.

**Comment by venraij** [comment]
What seems strange to me is that it's trying to go into @angular/core/core, that path does not exist.

**Comment by venraij** [comment]
It's also never imported like that. For example: 
![image](https://user-images.githubusercontent.com/45543766/141358852-f7072436-b0ab-4a34-9cdc-daf05de3a349.png)

**Comment by begandroide** [comment]
I understand, checking again.

**Comment by begandroide** [comment]
@venraij my apologizes, the angular version in my project is `11.2.4` and not `12`. So I will change the name of the issue. 

We have in roadmap upgrade to angular 12 so by the moments we can use `4.0.0`.

**Comment by venraij** [comment]
I've just locally tested in angular version 11.1.2, no issues at all again. Are you sure there isn't something else going wrong in your project?

![image](https://user-images.githubusercontent.com/45543766/141364086-8cce7b03-3706-4564-8b46-c25775f48ca7.png)

**Comment by venraij** [comment]
The 4.1.0 update doesn't involve big changes though besides the angular 13 upgrade. So there is not much reason to put a lot of time into it.

**Comment by begandroide** [comment]
@venraij did you install `ngx-mat-intl-tel-input`?  I didn't see it in your package.json

**Comment by venraij** [comment]
Yes, this is the testing project within the repo. I downgraded it to ng 11 for testing. It seems to work fine.

**Comment by begandroide** [comment]
ok, i will check and report here. Thanks for you help 😄

**Comment by begandroide** [comment]
@venraij maybe is related to nom registry, in StackBlitz i can reproduce the issue:

https://stackblitz.com/edit/angular-ivy-grjm68?file=package.json

**Comment by begandroide** [comment]
@venraij  the issue apparently is related with npm registry, I provide an StackBlitz repro link:

https://stackblitz.com/edit/angular-ivy-grjm68?file=src/app/app.component.ts

**Comment by venraij** [comment]
I think i've found the issue you were importing the module in the wrong way. This is what it should be:
![image](https://user-images.githubusercontent.com/45543766/141431917-9af5d50c-d5ea-42ae-9ecb-aec4eb03da78.png)

And this is what you had:
![image](https://user-images.githubusercontent.com/45543766/141431973-67051cc8-8910-4ad0-90d3-cfb99b31ad3b.png)

**Comment by venraij** [comment]
I've created a fix #136

**Comment by begandroide** [comment]
thanks you @venraij !! i will check when new version is available.
